events/dvg_venice.txt

```diff
@@ -0,0 +1,1879 @@
+﻿namespace = dvg_ven
+
+# starting patrician event
+dvg_ven.1 = {
```

Comment:
Small nitpick. But would be nice if Greek culture was also clickable in startup event. 
![image](https://github.com/vic3-divergences/divergences/assets/116951759/a87f1dd8-f001-453c-bef3-ef4cd2bc1921)

Reply:
done. see bruh commit.

Reply:
Its not working ingame. Still need to scope it in event
```
cu:greek= {
    save_scope_as = greek_culture
}
```

Reply:
okay, done. Anything else?